src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="/src/images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+
+      <div class="card__review-container">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <p class="card__reviews">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="card__price-container">
+        <span class="card__price-holder">
+          Цена:
+        </span>
+
+        <span class="card__price-value">
+          69 999 грн
+        </span>
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="card"
```

Comment:
please change `data-qa` attribute to the one asked in the task as attribute to the link